### DISCUSSION TITLE
cQube - Product vision discussion

### DISCUSSION THREAD

**Discussion by sukhpreetssekhon** [Sunbird-cQube/community#372] [General]
Hello Everyone,

Sharing the cQube product vision [doc](https://bit.ly/cqube_productvision) for your reference. This is a public document on which everyone can provide their comments. Please do share your comments by 4 October. This will be used as a starting point for evolving the cQube tech architecture further.

cc - @surendrasinghs @rahul101001000 @chakshugautam @Shruti3004 @sagarihanda @alok-os @ikpreet-singh

**Comment by sukhpreetssekhon** [comment]
The vision document is further updated with more clarity, additional section on what is cQube core and an updated functional block diagram.

cc - @surendrasinghs @rahul101001000 @ChakshuGautam @Shruti3004 @sagarihanda @alok-os @Ikpreet-Singh

**Reply by alok-os** [reply]
@sukhpreetsamagra - thanks for bringing out comprehensive thoughts on cQube. 
To start with, I suggest we focus on following few questions and align on them:
1. What is the problem we are trying to solve through cQube and Why
2. What are design considerations we will keep in mind
3. What is the positioning of cQube in context to similar capabilities available in the market
4. What broad capabilities cQube will enable aligned with pts above.

As we delve on the questions above; we should reflect on our experiences and learnings from relevant engagements - helping us to align with real problem and define sharper strategic cQube vision.

Once we are clear and aligned on points above then we can dive deeper into capability details, features and "How" part - architecture and design.

**Reply by surendrasinghs** [reply]
@sukhpreetsamagra I have added further notes to the cQube Vision document as per my understanding. I also notice there are a few open comments. Let's huddle and resolve for further clarity.
